keyboards/lets_split/rev1/info.json

```diff
@@ -23,6 +23,7 @@
     "layout_aliases": {
         "LAYOUT": "LAYOUT_ortho_4x12"
     },
+    "community_layout": ["ortho_4x12"],
```

Comment:
```suggestion
    "community_layouts": ["ortho_4x12"],
```

Reply:
thanks @fauxpark, corrected as of [c9255ef](https://github.com/qmk/qmk_firmware/pull/21855/commits/c9255ef41f81c6f656eb5b1dbab5df33d1045317)